an_aws_sam_imgup-compressor/src/index.js

```diff
@@ -18,18 +18,18 @@ exports.handler = async (event) => {
     // Compress the image to a 200x200 avatar square as a buffer, without stretching
     const compressedImageBuffer = await sharp(uncompressedImage.Body)
     .resize({ 
-        width: 200, 
         height: 200, 
         fit: 'contain'
     })
+    .png()
```

Comment:
Does conversion to `.png` have any other benefits (e.g. reduction of bandwidth) over other formats?

Reply:
Not particularly, but because I didn't strictly have access to the `content type` of the file in the `index.js` lambda function, as per https://github.com/lovell/sharp/issues/2232, I figured it would make sense to output the buffer to `png` and set the `content-type` as such when saving the file.

Reply:
Cool. Thanks. 👌